serverless/develop/serverless-event-sinks.adoc

```diff
@@ -6,7 +6,11 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-A sink is an addressable custom resource (CR) that can receive incoming events from other resources. Knative services, channels, and brokers are all examples of sinks.
+When you create an event source, you can specify a sink where events are sent to from the source. A sink is an addressable or a callable resource that can receive incoming events from other resources. Knative services, channels and brokers are all examples of sinks.
+
+Addressable objects receive and acknowledge an event delivered over HTTP to an address defined in their `status.address.url` field. As a special case, the core Kubernetes `Service` object also fulfils the addressable interface.
```

Comment:
I feel like a snob pointing this out, but according to the IBMSG, we should be using the US spelling of "fulfill" (with two Ls). https://docs.google.com/document/d/1wfjj6B_K-KxEHQ5Fp_WQmIaBUB-HH92y9_sBFsf8_Xg/edit

Sorry :X